<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImageAnalysisWidget</class>
 <widget class="QWidget" name="ImageAnalysisWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>642</width>
    <height>347</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widget_2" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="baseSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="2" column="1">
       <widget class="QComboBox" name="imageType">
        <item>
         <property name="text">
          <string>Binary</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Reduced binary</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Phase map</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Hybrid</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QWidget" name="widget_3" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="margin">
          <number>0</number>
         </property>
         <item>
          <widget class="QSpinBox" name="selectedU">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="selectedV">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_2">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Encoding:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QCheckBox" name="showKey">
        <property name="text">
         <string>Show key</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="2">
       <widget class="QCheckBox" name="showCrosshair">
        <property name="text">
         <string>Show crosshair</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="2">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Selected position:</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0" colspan="2">
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="1">
         <widget class="QSpinBox" name="cropTop">
          <property name="maximum">
           <number>1200</number>
          </property>
          <property name="singleStep">
           <number>10</number>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QSpinBox" name="cropLeft">
          <property name="maximum">
           <number>1600</number>
          </property>
          <property name="singleStep">
           <number>10</number>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QSpinBox" name="cropRight">
          <property name="maximum">
           <number>1600</number>
          </property>
          <property name="singleStep">
           <number>10</number>
          </property>
          <property name="value">
           <number>1600</number>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QSpinBox" name="cropBot">
          <property name="maximum">
           <number>1200</number>
          </property>
          <property name="singleStep">
           <number>10</number>
          </property>
          <property name="value">
           <number>1200</number>
          </property>
         </widget>
        </item>
        <item row="3" column="2">
         <widget class="QPushButton" name="pushButton">
          <property name="text">
           <string>Fit</string>
          </property>
          <property name="icon">
           <iconset resource="icons.qrc">
            <normaloff>:/oxygen/icons/oxygen/zoom-draw.png</normaloff>:/oxygen/icons/oxygen/zoom-draw.png</iconset>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Crop</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="ImageViewWidget" name="image" native="true">
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <zorder>widget_2</zorder>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ImageViewWidget</class>
   <extends>QWidget</extends>
   <header>gui/imageviewwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>imageType</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>ImageAnalysisWidget</receiver>
   <slot>setImageDisplayed(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>195</x>
     <y>87</y>
    </hint>
    <hint type="destinationlabel">
     <x>320</x>
     <y>173</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cropBot</sender>
   <signal>valueChanged(int)</signal>
   <receiver>ImageAnalysisWidget</receiver>
   <slot>cropBottomChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>128</x>
     <y>319</y>
    </hint>
    <hint type="destinationlabel">
     <x>320</x>
     <y>173</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cropLeft</sender>
   <signal>valueChanged(int)</signal>
   <receiver>ImageAnalysisWidget</receiver>
   <slot>cropLeftChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>47</x>
     <y>280</y>
    </hint>
    <hint type="destinationlabel">
     <x>320</x>
     <y>173</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cropRight</sender>
   <signal>valueChanged(int)</signal>
   <receiver>ImageAnalysisWidget</receiver>
   <slot>cropRightChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>214</x>
     <y>280</y>
    </hint>
    <hint type="destinationlabel">
     <x>320</x>
     <y>173</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cropTop</sender>
   <signal>valueChanged(int)</signal>
   <receiver>ImageAnalysisWidget</receiver>
   <slot>cropTopChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>128</x>
     <y>241</y>
    </hint>
    <hint type="destinationlabel">
     <x>320</x>
     <y>173</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>setImageDisplayed(int)</slot>
  <slot>cropTopChanged(int)</slot>
  <slot>cropBottomChanged(int)</slot>
  <slot>cropLeftChanged(int)</slot>
  <slot>cropRightChanged(int)</slot>
 </slots>
</ui>
